docs/snippets/install-kn.md

```diff
@@ -4,7 +4,7 @@
         For macOS, you can install `kn` by using [Homebrew](https://github.com/knative/homebrew-client){target=_blank}.
```

Comment:
Is this still relevant?

Reply:
I created an issue in the homebrew-client repo https://github.com/knative/homebrew-client/issues/32 and I will updated this PR.

you are correct we don't need this homebrew client repo

Reply:
I replaced the URL with the one of brew main page.